app/Services/InviteService.php

```diff
@@ -146,4 +147,9 @@ public function reInvite($id): array
             'token' => $token
         ];
     }
+
+    public function invitedUsers(): Collection
+     {
+         return Invite::latest()->withTrashed()->get();
```

Comment:
Why are we returning withThrashed ? we will also list the deleted invite users ?